docs/content/latest/explore/_index.md

```diff
@@ -181,10 +183,10 @@ Use [yb-ctl](../admin/yb-ctl) local cluster management utility to explore the co
     </a>
   </div>
   <div class="col-12 col-md-6 col-lg-12 col-xl-6">
-    <a class="section-link icon-offset" href="backup-restore/point-in-time-recovery-ysql/">
+    <a class="section-link icon-offset" href="cluster-management/point-in-time-recovery-ysql/">
       <div class="head">
         <img class="icon" src="/images/section_icons/explore/fault_tolerance.png" aria-hidden="true" />
-        <div class="title">Point-in-time recovery</div>
+        <div class="title">Cluster management</div>
       </div>
       <div class="body">
```

Comment:
Order of tiles needs fixes -
- Indexes and Constraints is missing
- Security goes before Observability
- remove Extensions and Sharding strategies

Reply:
Fixed